#2996 added top description in new ui

Patrykb0802 opened the pull request.

Patrykb0802 committed:
#2996 added top description in new ui

- Added dynamic refresh of top description on new UI

Patrykb0802 committed:
Added top description in new UI #2996

- Added custom stylesheet to new UI

Patrykb0802 committed:
Added top description in new UI #2996

- Added custom stylesheet to new UI

Patrykb0802 committed:
Added top description in new UI #2996

- Added custom stylesheet to new UI

Patrykb0802 committed:
Added top description in new UI #2996

Limraj committed:
Merge remote-tracking branch 'origin/release/2.8.0' into feature/#2996_Added_top_description_in_new_UI2

Limraj reviewed (commented):
We enter the message in the commit according to the following formula, in this order:
**#number_issue(1)** title_issue(2): description(3)

Patch with correct:
[#2996_Added_top_description_in_new_UI.patch](https://github.com/user-attachments/files/16901095/2996_Added_top_description_in_new_UI.patch)

Limraj committed:
#2996 Added top description in new UI:
- using the vue mechanism update instead of emit;
- fixed url for login and logout on new ui;

Limraj committed:
#2996 Added top description in new UI:
- moved customCss to systemSettings/index.js;
